wait_for_postgres.py

```diff
@@ -22,14 +22,14 @@
 def pg_isready(host, user, password, dbname):
     while time() - start_time < check_timeout:
         try:
-            conn = psycopg2.connect(**vars())
+            conn = psycopg2.connect(**config)
             logger.info("Postgres is ready! ✨ 💅")
             conn.close()
             return True
         except psycopg2.OperationalError:
             logger.info(
                 f"Postgres isn't ready. Waiting for {check_interval} "
-                f"{interval_unit}..."
+                f"{interval_unit}... using {config}"
             )
             sleep(check_interval)
 
```

Comment:
Why not using the `vars()` ? It was a good way to make it configurable.

If we don't use the vars anymore, we should remove it from the function parameters and change the call on line 42